diff --git a/index.ts b/index.ts
index 65260b6..8e15445 100644
--- a/index.ts
+++ b/index.ts
@@ -14,7 +14,7 @@ const app = express();
 setupExpress(app);
 setupRoutes(app);
 
-if (process.env.NODE_ENV === 'development') {
+if (process.env.NODE_ENV === 'production') {
   syncDatabase();
 }
 
diff --git a/src/config/database.ts b/src/config/database.ts
index 3828c77..043d528 100644
--- a/src/config/database.ts
+++ b/src/config/database.ts
@@ -36,10 +36,10 @@ const databaseConfig: DBConfigs = {
     },
   },
   production: {
-    username: process.env.DB_USERNAME,
-    password: process.env.DB_PASSWORD,
+    username: process.env.DB_AWS_USERNAME,
+    password: process.env.DB_AWS_PASSWORD,
     database: process.env.DB_DATABASE,
-    host: process.env.DB_HOST,
+    host: process.env.DB_AWS_HOST,
     port: process.env.DB_PORT,
     dialect: 'mysql',
     timezone: '+09:00',
diff --git a/src/loaders/syncDatabase.ts b/src/loaders/syncDatabase.ts
index 5101821..4290b02 100644
--- a/src/loaders/syncDatabase.ts
+++ b/src/loaders/syncDatabase.ts
@@ -1,9 +1,10 @@
 import db from '../models';
 import UserGroup from '../models/UserGroup';
 
-const forceSync = false;
+const forceSync = true;
 
 const syncDatabase = () => {
+  console.log('Starting database sync...');
   db.sequelize
     .sync({ force: forceSync, logging: false })
     .then(() => {
diff --git a/src/types/node.d.ts b/src/types/node.d.ts
index 57ecf5c..83bd258 100644
--- a/src/types/node.d.ts
+++ b/src/types/node.d.ts
@@ -6,7 +6,9 @@ declare namespace NodeJS {
     DB_USERNAME: string;
     DB_PASSWORD: string;
     DB_DEV_PASSWORD: string;
+    DB_AWS_USERNAME: string;
     DB_AWS_PASSWORD: string;
+    DB_AWS_HOST: string;
     DB_HOST: string;
     DB_DEV_HOST: string;
     DB_AWS_HOST: string;
